python/rikai/mixin.py

```diff
@@ -90,6 +90,51 @@ def __matmul__(self, style: Union[dict, "rikai.viz.Style"]) -> Drawable:
         return style(self)
 
 
+class ComposableDrawable:
+    """Mixin for a drawable that can be composed with others"""
+
+    @abstractmethod
+    def left_top(self):
```

Comment:
isnt this hard code the way the position? what if an appropriate label position would be bottom / center ?

Reply:
I kinda dont think this `left_top` and `aligned` belong to the same interface.

`left_top / position` seems to belong to `base_drawable`, while `aligned` is the one that attached to the `base_drawable` right?

Reply:
In my idea, `aligned` would apply some algorithm, `left_top` is an anchor that an algorithm may use. Because `left-top` is currently our only algorithm, so I only added this anchor, we may add other anchors later. I'm not sure if the anchor method would be useful in other places, so I did not add it in the base class.

Reply:
How about use the python function as the parameter.

In this case, `left_top` is a member of ComposableDrawable. Using python function as the parameter, we could separate left_top and the Mixin.